src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx

```diff
@@ -1,68 +0,0 @@
-import { captureException } from "@sentry/browser"
-import { verifyEmail } from "./verifyEmail"
-import { verifyID } from "./verifyID"
-import { useCallback, useState } from "react"
-import { useSystemContext } from "System/useSystemContext"
-import { Environment } from "relay-runtime"
-
-type verificationType = "ID" | "Email"
-
-const IDVerificationBlockingStates = ["passed", "failed", "watchlist_hit"]
```

Comment:
I neglected to point out that these `states` are never returned to the client. By my testing, when any of those [states](https://github.com/artsy/gravity/blob/main/app/models/domain/identity_verification.rb#L41) are set on the `identity_verification` the endpoint returns [Unprocessable entity](https://github.com/artsy/gravity/blob/4b98054c2159d7397d53eba0350e4d77af3cf22a/app/api/v1/identity_verifications_endpoint.rb#L73). Before the refactor, the client was not displaying this error. It now does.

<img width="601" alt="Screen Shot 2023-03-28 at 9 01 53 AM" src="https://user-images.githubusercontent.com/29984068/228246069-c7f81645-cbbf-4e22-a814-020b7dbf05b2.png">